content/ru/guide/v10/differences-to-react.md

```diff
@@ -37,8 +37,7 @@ Preact не претендует на то, чтобы быть повторно
 
 ## Отладочные сообщения и ошибки
 
-Our flexible architecture allows addons to enhance the Preact experience in any way they want. One of those addons is `preact/debug` which adds [helpful warnings and errors](https://preactjs.com/guide/v10/debugging) and attaches the [Preact Developer Tools](https://preactjs.github.io/preact-devtools/) browser extension, if installed. Those guide you when developing Preact applications and make it a lot easier to inspect what's going on. You can enable them by adding the relevant import statement:
-Наша гибкая архитектура позволяет аддонам улучшать работу Preact любым удобным для них способом. Одним из таких дополнений является `preact/debug`, который добавляет [полезные предупреждения и ошибки](https://preactjs.com/guide/v10/debugging) и присоединяет [Инструменты разработчика Preact](https://preactjs.github.io/preact-devtools/), если оответствующее расширение браузера установлено. Они помогут вам при разработке приложений Preact и значительно упрощают проверку того, что происходит. Вы можете включить их, добавив соответствующий оператор импорта:
+Наша гибкая архитектура позволяет аддонам улучшать работу Preact любым удобным для них способом. Одним из таких дополнений является `preact/debug`, который добавляет [полезные предупреждения и ошибки](/guide/v10/debugging) и присоединяет [Инструменты разработчика Preact](https://preactjs.github.io/preact-devtools/), если оответствующее расширение браузера установлено. Они помогут вам при разработке приложений Preact и значительно упрощают проверку того, что происходит. Вы можете включить их, добавив соответствующий оператор импорта:
```

Comment:
Looks to have been a translation bug, figured I'd strip the erroneous English paragraph while I was at it